saleor/order/capture_payment.py

```diff
@@ -0,0 +1,74 @@
+from decimal import Decimal
+from typing import Optional
+
+from saleor.account.models import User
```

Comment:
Absolute

Reply:
I'll put all of the logic under `actions.py` in the subsequent PR and fix import / remove underscores on the public interface.